ios-baseUITests/XCUIApplicationExtension.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  XCUIApplicationExtension.swift
+//  ios-baseUITests
+//
+//  Created by Germán Stábile on 2/13/20.
+//  Copyright © 2020 Rootstrap. All rights reserved.
+//
+
+import XCTest
+
+extension XCUIApplication {
+  func type(text: String, on fieldName: String, isSecure: Bool = false) {
+    let fields = isSecure ? secureTextFields : textFields
+    let field = fields[fieldName]
+    field.forceTap()
```

Comment:
Why `forceTap()` here and for the clear method below just `tap()`?

Reply:
I think tap here is enough, I was having a weird issue (still occurring in iOS 13) and was trying things, anyways I think forceTap can be the default so I'll update clear.